### PR TITLE
Consistent code/space

### PR DESCRIPTION
**Pull request by arxpoetica**

### CODE REVIEW

File: front-end/map-search-test/Readme.md [hunk 1]
```diff
@@ -5,25 +5,33 @@
 You've been given:
 
 1. `index.html`: The base HTML file with a full-page Google Map
-2. `main.css`: The base CSS file
-3. `main.js`: The base JS file
-4. `sample-data.js`: The sample data that should be searched. Each record has _at least_ an `id` and `name`.
+1. `main.css`: The base CSS file
```
Comment (mikefogg) [inline front-end/map-search-test/Readme.md:8]: @arxpoetica did you remove my 2,3,4 and replace them all with 1's!? haha

File: front-end/map-search-test/Readme.md [hunk 2]
```diff
@@ -5,25 +5,33 @@
...
+1. `main.js`: The base JS file
+1. `sample-data.js`: The sample data that should be searched. Each record has _at least_ an `id` and `name`.
 
 ## The Task
 
-Create a user interface that allows an end user to search through the provided sample data (found in `sample-data.js`) for results based on their `name`. When finding a search result, the end user should be able to select that result and have it's location displayed on the map. When clicking that map marker, you should display a modal (popup) that contains additional information about that location.  
+Create a user interface that implements the following:
+
+1. "Autocomplete" search results. Use the provided sample data (found in `sample-data.js`) to allows users to search for results based on the data `name` value.
```
Comment (mikefogg) [inline front-end/map-search-test/Readme.md]: "...allows users to *type in a search field* that returns results based on..." maybe? We ran into an issue (potentially) with the other one before where they didn't understand that they were supposed to create a text field.

File: front-end/map-search-test/Readme.md (hunk 1 above)
Comment (arxpoetica) [reply front-end/map-search-test/Readme.md:8]: @mikefogg yes. It's a markdown feature. You don't have to number them; they get automatically incremented in translation. From the documentation:

>It’s important to note that the actual numbers you use to mark the list have no effect on the HTML output Markdown produces.

(https://daringfireball.net/projects/markdown/syntax#list)

File: front-end/map-search-test/Readme.md (hunk 2 above)
Comment (arxpoetica) [reply front-end/map-search-test/Readme.md]: Good point. I'll clarify.

**Pull request merged**